src/Path.hs

```diff
@@ -231,6 +231,14 @@ relfile = qq mkRelFile
 --
 -- @x \<\/> $(mkAbsDir …)@
 --
+-- The following equalities hold for the special relative directory @.@:
+--
+-- @x \<\/> $(mkRelDir ".") == x@
```

Comment:
No, the core point here is to illustrate the property. Either TH splice or quasiquote should be enough. And whichever we chose should be used everywhere consistently.